VAMobile/src/components/CollapsibleView.tsx

```diff
@@ -33,10 +37,14 @@ export type CollapsibleViewProps = {
  */
 const CollapsibleView: FC<CollapsibleViewProps> = ({ text, contentInTextArea = true, showInTextArea = true, a11yHint, textColor, children, testID }) => {
   const theme = useTheme()
+  const { t } = useTranslation(NAMESPACE.COMMON)
   const [expanded, setExpanded] = useState(false)
 
   const onPress = (): void => {
     setExpanded(!expanded)
+    if (!expanded && text === t('loaGate.expandMsg')) {
```

Comment:
Ya, we can do better than this. We need to log all of these with a parameter not just a single one if we're going to keep using this component. I'll follow up with you.